docs/reference/glossary.asciidoc

```diff
@@ -105,6 +115,19 @@
   +
   See also <<glossary-routing,routing>>
 
+[[glossary-query]] query ::
+
+  Query refers to all queries which not only determine if a document matches, 
+  but also calculate how well the document matches. This calculation is refered 
+  to as scoring, hence these queries are also known as "scoring queries". 
+  A scoring query calculates how relevant each document is to the query, and assigns 
+  it a relevance _score, which is later used to sort matching documents by relevance. 
+  This concept of relevance is well suited to full-text search, where there is seldom a 
+  completely “correct” answer. These queries are more heavier than 
```

Comment:
Not sure if "more heavier" is correct (I'm not a native speaker) but it sounds like using "heavier" would be enough. Maybe "heavier" could also be exchanged for something that makes it clearer that we are talking about performance here.